whole-game.Rmd

```diff
@@ -42,11 +42,13 @@ It's meant to paint the Big Picture and suggest a workflow, before we descend in
 
 To keep the pace brisk, we exploit the modern conveniences in the devtools package and the RStudio IDE. In later chapters, we are more explicit about what those helpers are doing for us.
 
+We strongly suggest you follow along and create this toy package with us, however if you just want to read through it (or something goes awry halfway through!), completing it is not critical to understanding the remainder of the book.
```

Comment:
Hmmm, I think this is close but not quite right. If this chapter is impenetrable, I think the rest of the book is going to be rough sailing indeed. I think we should only make a more limited promise that this chapter is self-contained.

Reply:
Right, that's a good point!

Reply:
Something like: "We strongly suggest you follow along and create this toy package with us, however it is self-contained and completing it is not a requirement to advancing through the remainder of the book."?

Reply:
Pushed an alternate wording that is hopefully better.